### ISSUE TITLE
vtk was not found during compilation

### ISSUE THREAD

**Issue by karnuz** [PointCloudLibrary/pcl#5017]
Attempting to compile the example here : https://pcl.readthedocs.io/projects/tutorials/en/latest/bspline_fitting.html#bspline-fitting

I run into the following error 

```
CMake Error at /usr/local/share/pcl-1.12/PCLConfig.cmake:63 (message):
  visualization is required but vtk was not found
Call Stack (most recent call first):
  /usr/local/share/pcl-1.12/PCLConfig.cmake:356 (pcl_report_not_found)
  /usr/local/share/pcl-1.12/PCLConfig.cmake:539 (find_external_library)
  CMakeLists.txt:5 (find_package)
```

I do have vtk installed on my system and was able to compile project dependent on vtk.

**Comment by mvieth** [comment]
We need some more information, like:
What operating system do you use?
How did you install PCL (self compiled from source)?
Which version of PCL (the 1.12.0 release or the current master branch or some other commit)?
Did you install VTK and if yes, how and which version?

**Comment by karnuz** [comment]
On MacOS 11.5
Installed PCL and VTK using brew.
PCL version is 1.12
vtk version is 9.0

**Comment by mvieth** [comment]
Is there a vtk-config.cmake or VTKConfig.cmake on your computer? If you know where it is, you can try to set `VTK_DIR` to that path before `find_package(PCL)`

**Comment by karnuz** [comment]
Thanks for the suggestion. 
In the default VTK, there was no VTKConfig.cmake file. I built another VTK from source using CMake, with Qt enabled and set the VTK_DIR path to it and that  helped. The configuration now is Qt6 along with VTK 9.1. I was able to run an example (didn't include PCL) that uses this configuration.

However, with PCL, I now am running into the issue as mentioned in #1753. 

Do we know if there is a resolution to that ?
